youtube-comment-blacklist.user.js

```diff
@@ -15,7 +15,9 @@
 
 (() => {
 	"use strict";
-
+	
+ 	const removeThread = false; //if true, remove the whole thread, including replies
```

Comment:
I'll need to learn how to use the GreaseMonkey API to add settings so `removeThread` can be configured by the user

Reply:
Sorry, didn't mean to include the removeThread flag in this commit... different features, different commits.